### PR TITLE
Hongyli fix 4.8 failure

### PR DESCRIPTION
**Pull request by lihongyan1**
Fix UI failures caused by product change
Related to PR:
https://github.com/openshift/cucushift/pull/8602

v3 job for case OCP-21257
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/321158/

### CODE REVIEW

File: lib/rules/web/admin_console/4.8/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -354,15 +356,16 @@ list_alerts_by_filters_clear:
 list_alerts_by_filters:
   action: click_alert_filter_dropdown
   action: click_alert_filter_item
+  action: click_alert_filter_dropdown
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.8/monitoring_alerts.xyaml:359]: I can see it exist two line above.

Comment (lihongyan1) [reply lib/rules/web/admin_console/4.8/monitoring_alerts.xyaml:359]: > I can see it exist two line above.

One is open drop down, then select an item, the later one is closed drop down list

**Pull request merged**